src/registrar/tests/common.py

```diff
@@ -88,6 +101,226 @@ def send_email(self, *args, **kwargs):
+            last_name="{} last_name:{}".format(item_name, short_hand),
+            username="{} username:{}".format(item_name, short_hand),
+        )[0]
+        return user
+
+    def dummy_contact(self, item_name, short_hand):
+        """Creates a dummy contact object"""
+        contact = Contact.objects.get_or_create(
+            first_name="{} first_name:{}".format(item_name, short_hand),
+            last_name="{} last_name:{}".format(item_name, short_hand),
+            title="{} title:{}".format(item_name, short_hand),
+            email="{}testy@town.com".format(item_name),
+            phone="(555) 555 5555",
+        )[0]
+        return contact
+
+    def dummy_draft_domain(self, item_name):
+        """Creates a dummy draft domain object"""
+        return DraftDomain.objects.get_or_create(name="city{}.gov".format(item_name))[0]
+
+    def dummy_domain(self, item_name):
+        """Creates a dummy domain object"""
+        return Domain.objects.get_or_create(name="city{}.gov".format(item_name))[0]
+
+    def dummy_alt(self, item_name):
+        """Creates a dummy website object for alternates"""
+        return Website.objects.get_or_create(website="cityalt{}.gov".format(item_name))[
+            0
+        ]
+
+    def dummy_current(self, item_name):
+        """Creates a dummy website object for current"""
+        return Website.objects.get_or_create(website="city{}.com".format(item_name))[0]
+
+    def get_common_domain_arg_dictionary(
+        self,
+        item_name,
+        org_type="federal",
+        federal_type="executive",
+        purpose="Purpose of the site",
+    ):
+        """Generates a generic argument list for most domains"""
+        common_args = dict(
+            organization_type=org_type,
+            federal_type=federal_type,
+            purpose=purpose,
+            organization_name="{} organization".format(item_name),
+            address_line1="{} address_line1".format(item_name),
+            address_line2="{} address_line2".format(item_name),
+            is_policy_acknowledged=True,
+            state_territory="NY",
+            zipcode="10002",
+            type_of_work="e-Government",
+            anything_else="There is more",
+            authorizing_official=self.dummy_contact(item_name, "authorizing_official"),
+            submitter=self.dummy_contact(item_name, "submitter"),
+            creator=self.dummy_user(item_name, "creator"),
+        )
+        return common_args
+
+    def dummy_kwarg_boilerplate(
+        self,
+        domain_type,
+        item_name,
+        status,
+        org_type="federal",
+        federal_type="executive",
+        purpose="Purpose of the site",
+    ):
+        """
+        A helper function that returns premade kwargs for easily creating different domain object types.
+        There is a decent amount of boilerplate associated with
+        creating new domain objects (such as domain_application, or domain_information),
```

Comment:
this repeats a lot of what is up above. We don't need why it is being made. Just what does it do and what are the parameters and return values. 
With a comment this long I expect a return statement at the end of the docstring to indicate  "returns <type>" or an arugment breakdown. I know you mention it returns kwargs, which is just keyward args, but what should someone actually expect? It looks like it is a dictionary so maybe having a break down of what the return looks like an the values. A function that is going to return wildly different values like this ideally should indicate what a user should expect, because otherwise someone has to go line by line here to see what the format is for return.  I always like this example for docstrings although it is much simpler than your case:
def multiply (a,b)
   """
    Multiplies two numbers and returns the result.
 
    Args:
        a (int): The first number.
        b (int): The second number.
 
    Returns:
        int: The product of a and b.
    """